### DISCUSSION TITLE
📢 Hot Off the Press - Copilot Community Check-in

### DISCUSSION THREAD

**Discussion by Akash1134** [community/community#135224] [Copilot News and Announcements]
Hello! 👋 Just checking in for yet another riveting Copilot Community Check-In 🗞️
July was, of course, an action-packed month filled with groundbreaking Copilot releases and contributions from our ever-so-active community members.

## ✨ Community Champions: Shoutout 📢  to these members

Let's take a moment to celebrate the fantastic contributions of our community members who have gone above and beyond to make our Copilot community thrive!

- Kudos to @davevad93 for consistently being the top contributor in the community. Their detailed explanation, [outlined here](https://github.com/orgs/community/discussions/132079), has been a lifesaver for a member facing connectivity problems and offering [comprehensive troubleshooting tips](https://github.com/orgs/community/discussions/131604) to get users back on track. Contributions like these are a perfect example of exemplifying community spirit! 🌟
- A huge round of applause for @NirukshaSandeepa for helping a member connect their account to GitHub Copilot. By sharing a [concise yet thoughtful response](https://github.com/orgs/community/discussions/131649), they've made a real difference in resolving access issues for many users. ✨
- Big thanks to @lucasgabrielcastro for troubleshooting authentication problems in VSCode. They provided a practical fix to a rather complex issue, as discussed [in this thread](https://github.com/orgs/community/discussions/134876) -- such insights are invaluable! 🙌
- Cheers to @ikarusnc for a great Self-Solve :check and helping users keep their VSCode extensions up to date. Their [proactive advice](https://github.com/orgs/community/discussions/134281), ensures that everyone enjoys a smooth development experience. Great work! 👍

These community champions exemplify the spirit of collaboration and support that makes our community strong, turning challenges into opportunities for learning and growth. You, too, can make a difference by contributing your knowledge and insights.

> ✅ Mark as Answer: Don’t forget to click the **‘Mark as Answer**’ button when someone solves your question to easily help others who may have the same question ✅

## 🌍 Discover the Latest in Copilot: What's New?

Stay up-to-date with all things Copilot! We post monthly digests highlighting the latest features, updates, and releases. Check out our newest post, just published last week, to explore what's new and exciting.

### 👉 [Latest Copilot Updates](https://github.com/orgs/community/discussions/134273)

- 🔄 [Copilot Enterprise Mixed Licensing beta](https://github.blog/changelog/2024-07-26-copilot-enterprise-mixed-licensing-beta)
- 📊 [Enterprise Team Metrics Now Available on the Copilot Metrics API](https://github.blog/changelog/2024-08-08-enterprise-team-metrics-now-available-on-the-copilot-metrics-api/)
- 🤖 [GitHub Copilot Chat—Smarter Than Ever](https://github.blog/ai-and-ml/github-copilot/whats-new-with-github-copilot-july-2024/#github-copilot-chat-but-smarter)
- 🔧 [Improving GitHub Copilot Completions in VS Code for C++ Developers](https://github.blog/changelog/2024-08-01-improving-github-copilot-completions-in-vs-code-for-c-developers/)
- 📝 [Copilot Chat & Pull Request Summaries Powered by GPT-4o](https://github.blog/changelog/2024-07-31-github-copilot-chat-and-pull-request-summaries-are-now-powered-by-gpt-4o/)
- 🌐 [Copilot Network Requests Routed by Subscription](https://github.blog/changelog/2024-07-31-copilot-network-requests-are-now-routed-based-on-subscription/)
- 🔍 [GitHub Copilot knowledge bases available in Visual Studio](https://github.blog/changelog/2024-07-09-copilot-knowledge-bases-in-visual-studio-preview)

## 🛰️ The Software Extravaganza You’ve Been Waiting For - [GitHub Universe 2024](https://githubuniverse.com/) is Back! 🔙 

Oh joy, GitHub Universe is making a grand return 🤚 , and we’re just over the moon 🌕  to announce this in-person and virtual event on **October 29-30**. And guess what? It’s the 10th year, so it’s bound to be the grandest Universe ever! 🥳

🎥  [Prepare for GitHub Universe 2024: the ultimate ode to code and community](https://www.youtube.com/watch?v=hKgTC6d11w4)

- 📅 Mark the date → October 29-30
- 📍 Join us in San Francisco or tune in online.👉  3,500+ attendees → 150+ speakers → 100+ sessions → 35+ featured partners → all about AI, Security, and DevEx

**Grab your 💸 tickets cheaper before Sept 8! [Get your tickets here!](https://reg.githubuniverse.com/flow/github/universe24/Registration/login)**

What’s your favorite feature or trick that you love using with Copilot? We would love to hear your thoughts in the comments below. Or, if you’re feeling really adventurous, drop us a line about that one time Copilot saved your code from certain doom. Happy coding! 🚀

**Comment by Ali-is-cool** [comment]
copilot looks amazing
